### PR TITLE
Detailed Product Overview: Autopilot

### PR DESCRIPTION
**Pull request by wordsmither**
Closes #DOCO-926

## Summary

**[Autopilot](https://docs.pantheon.io/overview/custom-upstreams)** - New product overview page. One of many to come.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by wordsmither** [comment]
[Preview](https://pr-8250-documentation.appa.pantheon.site/overview/autopilot)

File: source/content/overview/autopilot.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- Tests the updates with automated visual regression testing (VRT)
+- Optionally deploys the updates
+
+![Autopilot Dashboard](../../images/autopilot/autopilot-sites-overview.png)
+
+</Tab>
+
+<Tab title="Features" id="features">
+
+* Automated Update Detection
```
Comment (kelly-mcgonnigle) [inline source/content/overview/autopilot.md:34]:
```suggestion
* Schedule updates Monthly, Weekly or Daily
```

File: source/content/overview/autopilot.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* Update Core CMS Upstreams
+* Hands-off Visual Regression testing
```
Comment (kelly-mcgonnigle) [inline source/content/overview/autopilot.md:36]:
```suggestion
* Hands-off visual regression testing (VRT)
```

File: source/content/overview/autopilot.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* Pre-configured workflows
+* Scheduled Updates on On-demand performance testing
```
Comment (kelly-mcgonnigle) [inline source/content/overview/autopilot.md:38]:
```suggestion

```

File: source/content/overview/autopilot.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+*View Recent Activity*
+
+![View Recent Activity](../../images/autopilot/autopilot-site-recent-activity.png)
+
+</Tab>
+
+<Tab title="Requirements" id="requirements">
+
+* Gold, Platinum or Diamond account.
+* If you're using Drupal, a [compatible version of Drush or Integrated Composer](https://docs.pantheon.io/guides/autopilot/autopilot-faq/#what-versions-of-drupal-are-supported-by-autopilot)
```
Comment (kelly-mcgonnigle) [inline source/content/overview/autopilot.md:59]:
```suggestion
* WordPress or Drupal site
* If you're using Drupal, a [compatible version of Drush or Integrated Composer](https://docs.pantheon.io/guides/autopilot/autopilot-faq/#what-versions-of-drupal-are-supported-by-autopilot)
```

File: source/content/overview/autopilot.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+* Multidev
```
Comment (kelly-mcgonnigle) [inline source/content/overview/autopilot.md:60]:
```suggestion

```

**Review by whitneymeredith — APPROVED** [review]
LGTM

**Pull request merged**